tests/engines/nwn2/roster.cpp

```diff
@@ -76,3 +76,18 @@ GTEST_TEST(NWN2Roster, rosterNames) {
 	EXPECT_STREQ(roster->getFirstRosterMember().c_str(), "ammon_jerro");
 	EXPECT_STREQ(roster->getNextRosterMember().c_str(),  "bishop");
 }
+
+/**
+ * Test the roster member settings in the ROSTER.rst file data.
+ */
+GTEST_TEST(NWN2Roster, rosterSettings) {
+	Common::ScopedPtr<TestRoster> roster(new TestRoster);
```

Comment:
Likewise, you can use std::unique_ptr here now. In fact, I'd prefer it. I'd like to migrate away from our ScopedPtr a bit.